fuel-core/src/database/coin.rs

```diff
@@ -90,16 +88,16 @@ impl Storage<UtxoId, Coin> for Database {
 }
 
 impl Database {
-    pub fn owned_coins(
+    pub fn owned_coins_utxos(
```

Comment:
why add "utxos"? These aren't utxos unless we filter out the spent coins.

Reply:
What is better `owned_coins_utxo_ids` or `owned_coins_ids`?=)

Reply:
`owned_coins_ids`